app/assets/javascripts/admin/addon/templates/customize-themes-show.hbs

```diff
@@ -290,7 +290,7 @@
     <a href={{previewUrl}} title={{i18n "admin.customize.explain_preview"}} rel="noopener noreferrer" target="_blank" class="btn btn-default">{{d-icon "desktop"}}{{i18n "admin.customize.theme.preview"}}</a>
     <a class="btn btn-default export" rel="noopener noreferrer" target="_blank" href={{downloadUrl}}>{{d-icon "download"}} {{i18n "admin.export_json.button_text"}}</a>
 
-    {{d-button action=(action "switchType") label="admin.customize.theme.convert" icon=convertIcon class="btn-default btn-normal" title=convertTooltip}}
+    {{d-button action=(action "switchType") label="admin.customize.theme.convert" icon=convertIcon class="btn-default btn-normal" disabled=model.default title=convertTooltip}}
```

Comment:
I'm sorry, I'm late on this, but... I think it would be better to remove the "Convert" button for the default theme. It's a very rare action, anyway.

Reply:
Done